ta/concurrent/Android.mk

```diff
@@ -0,0 +1,7 @@
+include $(CLEAR_VARS)
```

Comment:
The all-makefiles-under macro is doing the trick:

https://android.googlesource.com/platform/build/+/android-5.1.1_r37/core/definitions.mk#141